packages/web/bundlesize.prod.config.json

```diff
@@ -0,0 +1,8 @@
+{
+  "files": [
+    {
+      "path": "./build-ssr-production/server/chunks/chunk-*.js",
+      "maxSize": "30 kB"
```

Comment:
im not sure if this is is the maxSize for the full bundle or some of the chunks, but 30kb seems a bit low to me?

Reply:
its per bundle sadly, seems like there is no way to sum it all together 😢. maybe we look for another package

Reply:
ah i see. is it the regular or gzipped value? if gzipped i think 30 should be fine (there's one chunk that we have is at 23kb already so maybe need to bump to 40kb eventually)

Reply:
yeah in a way this is still useful cause its really just to catch libs getting in and making it 100+ kb